binding/web/template/src/porcupine.ts

```diff
@@ -565,11 +479,7 @@ export class Porcupine implements PorcupineEngine {
         contentAddress
       );
       try {
-        await pvFileOperationHelper({
-          command: 'file-save',
-          path: path,
-          content: content,
-        });
+        await storage.setItem(path, content);
```

Comment:
pedantic, but I think `setItem` should be `putItem` since we're talking about storage and that's what the db transaction is called

Reply:
As I understood it was done this way to match the `localStorage` API so it could be a drop-in fallback replacement

Reply:
ah okay, that makes sense